examples/bq_file_load_benchmark/benchmark_tools/tests/test_file_generator.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import json
+import os
+
+from google.cloud import bigquery
+from google.cloud import storage
+
+from bq_file_load_benchmark.benchmark_tools import file_generator
+
+
+class TestFileGenerator(object):
+    """Tests functionality of benchmark_tools.file_generator.FileGenerator.
+
+    Attributes:
+
+    """
+
+    def setup(self):
+        """Sets up resources for tests.
+        """
+       # create bucket
```

Comment:
Are these comments TODO?

Reply:
yes but I just added the logic for the tests